lib/zkevm_bridge.star

```diff
@@ -34,9 +36,9 @@ def start_bridge_ui(plan, args, config):
                 ),
             },
             env_vars={
-                "ETHEREUM_RPC_URL": config.l1_rpc_url,
-                "POLYGON_ZK_EVM_RPC_URL": config.zkevm_rpc_url,
-                "BRIDGE_API_URL": config.bridge_api_url,
+                "ETHEREUM_RPC_URL": bridge_ui_gateway_url + "/l1rpc",
+                "POLYGON_ZK_EVM_RPC_URL": bridge_ui_gateway_url + "/zkevmrpc",
+                "BRIDGE_API_URL": bridge_ui_gateway_url + "/bridgeapi",
```

Comment:
That setup is a bit weird, the bridge-ui redirects to the gateway and the gateway redirects to the bridge ui...